docs/sdk2/usage/TYPE_SYNC.md

```diff
@@ -183,6 +183,8 @@ __Note__ The statistics object contains the processing time of the last batch on
 **remove** the existing `fieldDefinition` and then **add** a new `fieldDefinition` with the new `FieldType`.
 
 2. The `fieldDefinition` with missing `fieldType` (is `null`) will not be synced.
+
+3. The `required` field of `FieldDefinition` cannot be updated!
```

Comment:
Is this sth new because of migration to sdk v2?

Reply:
No, the API doesn't provide an update action to change required field.

Reply:
[4525dca](https://github.com/commercetools/commercetools-sync-java/pull/1013/commits/4525dca84db9d874c237d2eab827d8df3cd3b25e)